libs/component-sample/src/lib/feature/search-result-list-sample/search-result-list-sample.component.html

```diff
@@ -8,6 +8,15 @@
   </sds-search-result-list>
   <br>
   <br>
+  <sds-search-result-list [model]="compare" [divider]=none>
```

Comment:
The comparison view should be part of the main results list items and not a separate list of items.